models/public/vehicle-reid-0001/vehicle-reid-0001.md

```diff
@@ -0,0 +1,76 @@
+# vehicle-reid-0001
```

Comment:
possibly, it will be better to name model `vehicle-reidentifiction-0001` for consistency with `person-reidentification` models

Reply:
I'm not sure that it's possible because the name of the model is set in roadmap. Moreover, initially I set the name as you said and than changed according official name.

Reply:
if it is from roadmap, than ok